LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2022 Plygrynd Jynkyrd
+Copyright (c) 2023 Plygrynd Jynkyrd
```

Comment:
we don't allow odd years in production, please change it for the next even number